### PR TITLE
feat(compare): compare two csv files with compare section

### PR DESCRIPTION
**Pull request by b5**
builds on #469. I'll rebase after it's merged

Things to check & fix:
* [ ] centering / layout for diff display table
* [ ] should show "these files are equal" when diff script contains no changes. Can look at diff stat for this

### CODE REVIEW

File: app/components/compare/CompareSidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      </div>
+    </div>
+  )
+}
+
+const mapStateToProps = (state: Store, ownProps: CompareSidebarProps) => {
+  return ownProps
+}
+
+// TODO (b5) - this component doesn't need to be a container. Just feed it the right data
```
Comment (b5) [inline app/components/compare/CompareSidebar.tsx]: Stray comment

File: app/components/compare/TableDiff.tsx [hunk 2]
```diff
@@ -1,107 +1,111 @@
...
-              if (open) {
+      <div id='table-diff-container'>
+        <table>
+          {data.schema && <TableDiffSchemaHead data={schema} />}
+          <tbody>
+            {data.diff.map((p, i) => {
+              if (Array.isArray(p[0])) {
+                count = 1
+                // leftCount = p[0][0][0]
+                // rightCount = p[0][1][0]
```
Comment (b5) [inline app/components/compare/TableDiff.tsx]: Stray Comments

File: app/components/compare/TableDiff.tsx [hunk 3]
```diff
@@ -1,107 +1,111 @@
-import React, { useState } from 'react'
+import React from 'react'
+
+import TableDiffSchemaHead from './TableDiffSchemaHead'
+import DiffStat from './DiffStat'
 
 interface TableDiffProps {
   data: any
```
Comment (b5) [inline app/components/compare/TableDiff.tsx]: Ideally this would be a type definition for a diff script. One day...

Comment (ramfox) [reply app/components/compare/TableDiff.tsx]: yez

**Pull request merged**